### DISCUSSION TITLE
Academic questions about light curves

### DISCUSSION THREAD

**Discussion by hiraga13** [phoebe-project/phoebe2#532] [General]
A question from Physics Alumna:
I had a passion for astronomy especially eclipse binaries couple months ago.
I have tried to work on IRAF to find some light curves about time of minimum of some eclipse binaries and got something like the picture below.
But there are some problems that I couldn't get while working on light curves even though reading some relate articles:
1. Why do we have to research on time of minima of eclipse binaries? Aren't their light curves what the final things we get? And what the conclusion we get after get their light curves
2. I've realized that SIMBAD (from http://simbad.u-strasbg.fr/simbad/) and relate links have a lot of light curves data? Isn't it really meaningless when doing a search on time of minima of eclipse binaries?
3. Normally, what will we deduce after got those binaries' light curves like this photo below?
Thank you.
![IMG_20210920_211950.jpg](https://user-images.githubusercontent.com/87889357/134029959-e39465ca-a82f-4066-900b-45f37660b565.jpg)

**Comment by bpablo** [comment]
Thank you for your inquiries. I will try to answer your questions below. 

1.) The reason we care about times of minima (TOMs) is that binary stars are not always constant. They location of eclipses can change due to a myriad of factors. Measuring these changes can give us a lot of interesting information about the system and it’s evolution as well as letting us know if there are unseen components.  For a detailed explanation, you can go here:

https://www.aavso.org/sites/default/files/education/vsa/Chapter13.pdf

This shows that Times of Minima can also be used in the analysis of other variable stars as well, though binary systems are likely the most common use. This chapter is also part of a larger online series dedicated to variable star astronomy. You can find the whole collection here:

https://www.aavso.org/education/vsa

2.) There are indeed multiple sources of light curves from surveys across astronomy. Just to name a few that have online data there is ASAS, ASASSN, TESS, Kepler and multiple others. Some of these are very useful for Times of Minima (i.e. Kepler and possibly TESS) while others are not. That being said, Eclipsing binary research is much broader than simply TOMs. Just because you can’t use the data explicitly for TOMs, that doesn’t mean it isn’t valuable. There are multiple uses for the data such as finding and analyzing a phased light curve. 

3.) This question is almost impossible to answer in the context of a reply on github. If you are asking what you can determine from an eclipsing binary light curve, then the answer is several relative parameters about the system, such as the temperature ratio of the components, the eccentricity of the orbit. If you are asking what you can do with a full binary solution (a light curve and radial velocity curve), then you can determine many fundamental parameters of the binary system e.g. masses and radii which are very difficult to determine in any other way. This subject is not small. Here is a short lecture on the topic:

https://sites.astro.caltech.edu/~george/ay20/Ay20-Lec4x.pdf

https://www.atnf.csiro.au/outreach//education/senior/astrophysics/binary_intro.html

I hope this helps. If you have further questions I would encourage you to visit a site like www.aavso.org. You have to make a free account, but once you do, they have a forum where you can ask such questions of like-minded individuals.

**Reply by hiraga13** [reply]
It was so awesome of you. This helped me a lot. I'll follow aavso forum to get more information of this topic. Thank you again for your enthusiasm.
